src/res/translation/translation_pl_PL.ts

```diff
@@ -2973,7 +2973,7 @@
     <message>
         <location filename="../../global.h" line="237"/>
         <source>For more information use the &quot;What&apos;s This&quot; help (help menu, right mouse button or Shift+F1)</source>
-        <translation>Aby uzyskać więcej informacji użyj &quot;Co to&quot; (Pomoc, prawy przycisk myszy lub Shift+F1)</translation>
+        <translation>Aby uzyskać więcej informacji użyj „Co to jest” (z menu Pomoc lub prawy przycisk myszy lub Shift+F1)</translation>
```

Comment:
I assume you have verified that these quotes work properly in the UI?

Reply:
Yes. I have been using those for a long in another Qt project I'm translating and it appears correctly under any OS-es with Polish localization.